boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w.dts

```diff
@@ -33,6 +34,23 @@
 	pinctrl-names = "default";
 };
 
+&uart2 {
+	status = "okay";
+	current-speed = <115200>;
+
+	/* HCI-UART pins*/
+	pinctrl-0 = <&p3_1_scb2_uart_tx &p3_0_scb2_uart_rx &p3_2_scb2_uart_rts &p3_3_scb2_uart_cts>;
+	pinctrl-names = "default";
+
+	bt-hci {
+		status = "okay";
+		compatible = "infineon,cyw43xxx-bt-hci";
+		bt-reg-on-gpios = <&gpio_prt3 4 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
```

Comment:
I think that `GPIO_PULL_UP` doesn't do anything here, it gets configured at the same time as when pin is set as output. If anything it's either going to be ignored or waste current on the internal pullup when the pin is eventually driven low, which is exactly when you do NOT want it to do that because you are trying to go in low power. Just `GPIO_ACTIVE_HIGH`.

Reply:
updated.